NightscoutServiceKit/Extensions/NightscoutEntry.swift

```diff
@@ -0,0 +1,47 @@
+//
+//  NightscoutEntry.swift
+//  NightscoutServiceKit
+//
+//  Created by Darin Krauss on 10/13/19.
+//  Copyright © 2019 LoopKit Authors. All rights reserved.
+//
+
+import LoopKit
+import NightscoutUploadKit
+
+extension NightscoutEntry {
+
+    convenience init(storedGlucoseSample: StoredGlucoseSample) {
+        self.init(
+            glucose: Int(storedGlucoseSample.quantity.doubleValue(for: .milligramsPerDeciliter)),
+            timestamp: storedGlucoseSample.startDate,
+            device: "loop://\(UIDevice.current.name)",
+            glucoseType: .Sensor
+//            direction: direction      // TODO: What to do here?
```

Comment:
[nit] is this needed?

Reply:
This is one of those things the old Nightscout code attempted to infer from separate data. Will create a JIRA ticket to explicitly add sensor direction (if available) to the StoreGlucoseSample.

Reply:
https://tidepool.atlassian.net/browse/LOOP-970